test/software/amazon/ion/TimestampTest.java

```diff
@@ -2870,6 +2871,25 @@ public void testGetMillisWithLargeScaleBigDecimal()
         Timestamp.forMillis(LARGE_SCALE_DECIMAL, PST_OFFSET).getMillis();
     }
 
+    @Test
+    public void testMillisIsIndepedentOfOffset() throws ParseException {
+        DateFormat format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ");
+        Date date = format.parse("2010-06-29T23:20:00+0000");
+
+        Calendar cal1 = Calendar.getInstance();
+        cal1.setTimeZone(TimeZone.getTimeZone("GMT+00:00"));
+        cal1.setTime(date);
+        Timestamp t1 = Timestamp.forCalendar(cal1);
+
+        Calendar cal2 = Calendar.getInstance();
+        cal2.setTimeZone(TimeZone.getTimeZone("GMT+02:00"));
+        cal2.setTime(date);
+        Timestamp t2 = Timestamp.forCalendar(cal2);
+
+        assertEquals(t1.getMillis(), t2.getMillis());
+        assertEquals(t1.getDecimalMillis(), t2.getDecimalMillis());
```

Comment:
I think we should assert two more things:
1. The value of `getMillis()` and `getDecimalMillis()` is the same.
2. This is the *correct* value (i.e. it's the same as `cal1.getTimeInMillis()`).

Reply:
sure thing--done